docs/administration/retention-policies/index.md

```diff
@@ -139,7 +139,7 @@ In Octopus Server, you can keep all, or select a number of releases to keep.
 Alternatively, you can specify a number of days worth of releases and files to keep.
 
 :::hint
-Octopus Cloud instances are subject to [storage limits](https://octopus.com/pricing/faq#cloud-storage-limits). For releases, that limit is set to keep the last 30 days worth.
+Octopus Cloud instances are subject to [storage limits](https://octopus.com/pricing/faq#cloud-storage-limits). Paid instances have no retention limit, free instances are limited to keep no more than the last 30 days worth.
```

Comment:
I think we should add something about the default retention which we set even for paying customers.

Reply:
:+1: I've added that the default is also applied, but can be changed.